Homework8_2/RealEstateObject.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Homework8_2
+{
+    public class RealEstateObject
+    {
+        private string? _address;
+        private double _square;
+        private int _price;
+        public string? address
+        {
+            get => _address;
+            set
+            {
+                if (value.Equals(string.Empty))
+                    throw new RealEstateObjectExceptions(value);
+                else
+                    _address = value;
+            }
+        }
+        public double square
+        {
+            get => _square;
+            set
+            {
+                if (value < 20)
+                    throw new RealEstateObjectExceptions(value);
+                else
+                    _square = value;
+            }
+        }
+        public int price
```

Comment:
You can all create a method to generate the exception

Reply:
done